core/trino-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -1502,11 +1502,11 @@ export class QueryDetail extends React.Component {
                                             Cumulative User Memory
                                         </td>
                                         <td className="info-text">
-                                            {formatDataSizeBytes(query.queryStats.cumulativeUserMemory / 1000.0) + " seconds"}
+                                            {formatDataSize(query.queryStats.cumulativeUserMemory / 1000.0) + "*seconds"}
                                         </td>
                                         {taskRetriesEnabled &&
                                         <td className="info-failed">
-                                            {formatDataSizeBytes(query.queryStats.failedCumulativeUserMemory / 1000.0) + " seconds"}
+                                            {formatDataSize(query.queryStats.failedCumulativeUserMemory / 1000.0) + "*seconds"}
```

Comment:
`MB*s` ? (`MBs` would be correct, but not clear)

Reply:
I started with MBs but yeah, I think it is not clear. 
Elsewhere we use full `seconds` (not an abbreviation), so I guess this is fine.